Gemfile

```diff
@@ -2,5 +2,6 @@
 
 source 'https://rubygems.org'
 
-# Specify your gem's dependencies in simple_segment.gemspec
+# Specify your gem's dependencies in rudder_analytics_sync.gemspec
+gem "debug"
```

Comment:
We probably don't want to keep it as a depencency

Reply:
This is okay. We haven't specified it under the `gemspec` file.